samples/weather_function/readme.md

```diff
@@ -0,0 +1,36 @@
+# Weather Function Sample
+
+This is a Serverless Function that fetches the current weather information. Part of the Function's output includes HTTP response headers, and the HTTP response body. This sample is designed to be deployed on OpenFaaS via a Docker image.
+
+This document assumes that you are using a PC running Debian (v9 or later), Ubuntu (v14.04 or later), or Linux Mint (v17 or later).
```

Comment:
Since it is docker, why is it "needed" to have a Debian, Ubuntu or Linux Mint OS? 🤔